src/test/resources/org/opentripplanner/middleware/bugsnagEvent.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "5ee8f475005b805e58010000",
```

Comment:
Please use fake values for IDs in this file.

Reply:
Updated.